src/lib.rs

```diff
@@ -12,6 +12,7 @@ pub mod coprocessor;
 pub mod eval;
 pub mod expr;
 pub mod field;
+pub mod file_map;
```

Comment:
Do we use the file map anywhere outside of public parameter caching? If not then it should be an internal module in `public_parameters/mod.rs`.

Reply:
I think you're right, I put it back in `public_parameters` module.